src/services/billing/cost-management/budget/modules/budget-list/BudgetListCard.vue

```diff
@@ -140,12 +137,32 @@ export default {
 
         const formatTime = (time: string) => i18nDayjs.value(time).format('MMMYYYY');
 
+        const getParentNameOfProject = async () => {
```

Comment:
여긴 함수명 `getParentProjectGroupName`이 어떨까요?
그리고 `getParentNameOfProject`와 `getParentNameOfProjectGroup` 함수를 하나로 합치는 것도 괜찮을 듯요.
아 그리고 api 사용할 땐 try catch로 감싸주세요!

```suggestion
        const getParentProjectGroupName = async (projectId?: string, projectGroupId?: string) => {
            try {
                if (projectId) { ... }
                else if (projectGroupId) { ... }
            } catch (e) {
                ...
                return undefined;
            }
        }
```